buildpack/buildpack.go

```diff
@@ -18,14 +18,17 @@ package buildpack
 
 import (
 	"fmt"
-	"path/filepath"
-
 	"github.com/buildpack/libbuildpack/buildpack"
 	"github.com/cloudfoundry/libcfbuildpack/logger"
 	"github.com/mitchellh/mapstructure"
+	"path/filepath"
 )
 
-const cacheRoot = "dependency-cache"
+const (
+	cacheRoot            = "dependency-cache"
+	DEPENDENCIES         = "dependencies"
```

Comment:
These constants should be consistently named.  Idiomatic Go dictates that the names be standard camel case with first character casing dependent on visibility.  I believe that this means `dependencies`, and `defaultVersions` (see next comment).